tiling.js

```diff
@@ -1121,6 +1165,32 @@ border-radius: ${borderWidth}px;
         }
     }
 
+    updateWindowPositionBar() {
+        // if pref show-window-position-bar is set to false, hide
+        if (!prefs.show_window_position_bar) {
+            this.windowPositionBar.hide();
+            return;
+        }
+        
+        // number of columns (a column have one or more windows)
+        let cols = this.length;
+        if (cols <= 1) {
+            this.windowPositionBar.hide();
+            return;
+        } else {
+            this.windowPositionBar.show();
+        }
+        
+        let width = this.monitor.width;
```

Comment:
```suggestion
        let width = this.monitor.width;
        this.windowPositionBarBackdrop.width = this.monitor.width;
```

Also update the backdrop width here. To be safe.

Reply:
I'll see if there's a better way to implement this (e.g. set width of backdrop on monitor.width change) - and/or can add this.  Actually I'll add this now and then look for an improvement.

Reply:
Implemented in [4d0eab4](https://github.com/paperwm/PaperWM/pull/476/commits/4d0eab4d073588bcaba8a434e3eb88c9866541af).